### PR TITLE
HEL-263 | Configure cleanup job in production

### PR DESCRIPTION
**Pull request by klempine**
I added the necessary configuration to run the cleanup commands in production
using a Cronjob. The job runs daily at midnight.

### CODE REVIEW

File: .github/workflows/production.yml [hunk 1]
```diff
@@ -83,3 +83,17 @@ jobs:
...
           VAULT_KV_SECRET_MOUNT_POINT: stable
+
+      - name: Deploy Cronjob to send removal notifications and clean old data
+        uses: City-of-Helsinki/setup-cronjob-action@main
+        with:
+          name: send-reminders-and-clean-old-data-cronjob
+          image_repository: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
+          image_tag: ${{ github.sha }}
+          kubeconfig_raw: ${{ env.KUBECONFIG_RAW }}
+          target_namespace: ${{ secrets.K8S_NAMESPACE_STAGING }}
```
Comment (Hi-Fi) [inline .github/workflows/production.yml:94]: STABLE

File: .github/workflows/production.yml [hunk 2]
```diff
@@ -83,3 +83,17 @@ jobs:
...
+          schedule: "0 0 * * *" # Daily at midnight
```
Comment (Hi-Fi) [inline .github/workflows/production.yml:95]: Exactly at midnight is a bit bad... some minutes later probably?

**Review by Hi-Fi — CHANGES_REQUESTED** [review]

File: .github/workflows/production.yml (hunk 1 above)
Comment (klempine) [reply .github/workflows/production.yml:94]: 🤦 Copypasted the configuration from staging, forgot to change that.

File: .github/workflows/production.yml (hunk 2 above)
Comment (klempine) [reply .github/workflows/production.yml:95]: Sure, why not.

File: .github/workflows/production.yml (hunk 1 above)
Comment (klempine) [reply .github/workflows/production.yml]: Fixed it now.

File: .github/workflows/production.yml (hunk 2 above)
Comment (klempine) [reply .github/workflows/production.yml]: Changed it to 00:15

**Review by Hi-Fi — APPROVED** [review]

**Pull request merged**